### ISSUE TITLE
Option No doesn't work

### ISSUE THREAD

**Issue by odoe** [dojo/cli-upgrade-app#30]
I was upgrading an app with `dojo upgrade app` after installing the [instructions here](https://github.com/dojo/framework/blob/master/docs/V5-Migration-Guide.md) and got the message to try a dry run first, so typed `n` and it still went ahead with the upgrade :)

![Screen Shot 2019-04-09 at 10 44 47 AM](https://user-images.githubusercontent.com/206462/55822767-2b16b680-5ab5-11e9-89ad-a09e7410869f.png)

Wasn't a big deal, as I had just cloned the repo, but still an issue I thought should be brought up.

Tried a couple of more times in case the _case_ mattered and `N` and no args, just enter all behave the same.

OS: Mac
Node: 10.9.0
NPM: 6.9.0

**Comment by nicknisi** [comment]
@odoe Thanks for reporting this! I took a look and added a fix in #32.

**Comment by schontz** [comment]
Not sure when this fix is supposed to be available, but still seeing the issue in v.5.0.0.

OS: Mac

```
$ dojo upgrade
? This command will irreversibly modify files. Are you sure you want to run the upgrade?
Use the --dry option first if in doubt No

Running version 3 upgrade scripts.


ℹ Running transform: Convert to framework paths


✔  transform complete.
0 Errors 24 OK 0 Skipped 23 Unchanged
...
```
